src/cephadm/cephadm.py

```diff
@@ -4329,6 +4331,33 @@ def command_gather_facts(ctx: CephadmContext) -> None:
 ##################################
 
 
+def command_custom_logrotate(ctx: CephadmContext) -> int:
+    """Writes out custom logrotate config for either cephadm.log or a cluster's cluster logs"""
+    custom_template: str = ''
+    if 'logrotate_config_file' in ctx and ctx.logrotate_config_file:
```

Comment:
super nitpicky and I don't really care if you change it, but I do generally suggest getting/checking the attribute in one action using `getattr`. Like:
```python
if  getattr(ctx, 'logrotate_config_file', None):
   ...
```
Also since new versions of ceph require python versions newer than 3.8 you can also choose to assign it to a variable on the same line using the 'walrus operator' too:
```python
if config_file := getattr(ctx, 'logrotate_config_file', None):
   ...
```
for example.

Maybe consider either option for the next time you're writing code like this.